### PR TITLE
added four differential equations

### PR DESCRIPTION
**Pull request by HH3117**

### CODE REVIEW

File: benchmarks/StiffODE/allen_cahn_wpd.jmd [hunk 1]
```diff
@@ -0,0 +1,230 @@
+---
+title: Benchmark for several DiffEqs
```
Comment (ChrisRackauckas) [inline benchmarks/StiffODE/allen_cahn_wpd.jmd]: Make the title be for the equation

File: benchmarks/StiffODE/allen_cahn_wpd.jmd [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+@time wp6 = WorkPrecisionSet(prob,abstols,reltols,setups;
+                            print_names=true, names=labels,
+                            numruns=5, error_estimate=:l2,
+                            save_everystep=false, appxsol=test_sol, maxiters=Int(1e5));
+
+plot(wp6, label=labels, markershape=:auto, title="Between family, medium order")
+```
+
+## Environment information
+
```
Comment (ChrisRackauckas) [inline benchmarks/StiffODE/allen_cahn_wpd.jmd]: See the other .jmds for the updated way to do this.

Comment (HH3117) [reply benchmarks/StiffODE/allen_cahn_wpd.jmd]: Do u mean put this instead?
```julia{echo=false}
using DiffEqBenchmarks
DiffEqBenchmarks.bench_footer(WEAVE_ARGS[:folder],WEAVE_ARGS[:file])
```

Comment (ChrisRackauckas) [reply benchmarks/StiffODE/allen_cahn_wpd.jmd]: yup

**Pull request merged**